ILRepack/RepackImporter.cs

```diff
@@ -120,6 +120,11 @@ public MethodReference Import(MethodReference reference, IGenericParameterProvid
         public TypeDefinition Import(TypeDefinition type, Collection<TypeDefinition> col, bool internalize)
         {
             _logger.Verbose("- Importing " + type);
+            bool hasFilter = String.IsNullOrEmpty(_options.RepackDropAttribute) == false;
+            if (hasFilter && ShouldDrop(type))
```

Comment:
considering moving `hasFilter` check inside `ShouldDrop`

Reply:
I thought placing the result (hasFilter) outside of `ShouldDrop` can save some calculations since it would not change during merging.

For performance considerations, maybe it is better to escalate it to a read-only local variable.
For code clearance, placing it into `ShouldDrop` might be reasonable.

What kind of approach would you prefer?

Reply:
Or shall I refactor both `hasFilter` and `ShouldDrop` into a new type, for instance `RepackFilter`?

Reply:
Appreciate the concern for performance, but I prefer code clarity over performance speculation (IMO perf difference won't be noticeable, partly because inlining and partly because it's negligible compared to the rest of the processing here)

Reply:
OK, moved `hasFilter` check into `ShouldDrop`